agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/GeneDiseaseAnnotationService.java

```diff
@@ -87,6 +91,18 @@ public List<GeneDiseaseAnnotation> getFiltered() {
 	public Map<Gene, List<DiseaseAnnotation>> getOrthologousGeneDiseaseAnnotations(Map<String, Pair<Gene, ArrayList<DiseaseAnnotation>>> geneMap) {
 		ProcessDisplayHelper display = new ProcessDisplayHelper(10000);
 
+		String orthoCacheFileName = "gene_disease_via_orthology_annotation.json.gz";
+		
+		HashMap<Gene, List<DiseaseAnnotation>> newDAMap = readFromCache(orthoCacheFileName, HashMap.class);
+		
+		if(newDAMap == null) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChBkMjViNzgxYzgzNmJmN2FlEgpjaGVja3N0eWxl -->